cmd/kubelet/app/server.go

```diff
@@ -553,7 +553,7 @@ func createClientConfig(s *options.KubeletServer) (*restclient.Config, error) {
 		if err != nil {
 			return nil, err
 		}
-		authConfig.Host = s.APIServerList[0]
```

Comment:
i agree with you, in previous versions of the patch only Hosts were used (instead of AlternateHosts and Host) but couple of reviewers suggested to reduce scope of this change and make it additive